### PR TITLE
Update NTP wallpapers (Fall 2019)

### PR DESCRIPTION
**Pull request by cezaraugusto**
Close https://github.com/brave/brave-browser/issues/6904

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
* New tab page images should be updated.
* Author attribution should match PDF file located in the referenced issue.

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: components/brave_new_tab_ui/data/LICENSE [hunk 1]
```diff
@@ -1,63 +1,59 @@
...
-License: used with permission
+dc-cavalleri.webp:
+URL: Contributor sent the hi-res version through email
+License: Used with permission
 
-ntp-9.webp:
-URL: https://www.instagram.com/p/BI7w_dFAlR0/
-License: used with permission
+joe-gardner.webp:
+URL: Contributor sent the hi-res version through email
```
Comment (fmarier) [inline components/brave_new_tab_ui/data/LICENSE]: According to https://github.com/brave/brave-browser/issues/6904#issuecomment-559242794, that should be https://unsplash.com/photos/4xv3lqnanYc and the Unlicense license.

File: components/brave_new_tab_ui/data/LICENSE [hunk 2]
```diff
@@ -1,63 +1,59 @@
...
-License: used with permission
+louis-kim.webp:
+URL: Contributor sent the hi-res version through email
+License: Used with permission
 
-ntp-11.webp:
-URL: https://www.instagram.com/p/BNyaAuFBiSt/
-License: used with permission
+matt-palmer.webp:
+URL: Contributor sent the hi-res version through email
```
Comment (fmarier) [inline components/brave_new_tab_ui/data/LICENSE]: According to https://github.com/brave/brave-browser/issues/6904#issuecomment-559242794, that should be https://unsplash.com/photos/veMnvjmfoxw and the Unlicense license.

**Comment by bsclifton** [comment]
@fmarier ready for re-review 😄

**Comment by bsclifton** [comment]
@cezaraugusto even though we merged, would you be able to answer questions by @petemill?

I checked myself on Nightly and the images are not horrible, but there are some quality issues (compression artifacts, pixelation)

**Pull request merged**